contentpacks/khanacademy.py

```diff
@@ -908,6 +909,12 @@ def _download_image_urls(url):
     item_data = localize_content_links(item_data)
     item_data = localize_graphie_urls(item_data)
 
+    # Validate assessment item content.
+    for k, v in yaml.load(item_data["item_data"]).items():
+        if k == "question":
+            if not v.get("content"):
```

Comment:
Please log some messages here to help us trace the missing contents on KA's API.